x-pack/plugins/spaces/server/lib/space_schema.test.ts

```diff
@@ -234,7 +237,7 @@ describe('#imageUrl', () => {
 describe('#solution', () => {
   it('should throw error if solution is defined in serverless offering', () => {
     expect(() =>
-      spaceSchema.validate({ ...defaultProperties, solution: 'search' }, { serverless: true })
+      getSpaceSchema(true).validate({ ...defaultProperties, solution: 'search' })
```

Comment:
nit: should we add a test case for `getSpaceSchema(false)` somewhere too?

Reply:
There is [a couple of them](https://github.com/elastic/kibana/pull/183986/files#diff-93f4f463a8823ea9deeb6ad16e11293f98ba4c4343414a4ab2b161563e49a87eR250), I just saved the base schema into variably on the top, so probably missed them

`const spaceSchema = getSpaceSchema(false);`

Reply:
I will make it more readable though